cms/static/js/certificates/models/certificate.js

```diff
@@ -20,9 +20,25 @@ function (_, str, Backbone, BackboneRelational, BackboneAssociations, gettext, C
         defaults: {
             name: 'Name of the certificate',
             description: 'Description of the certificate',
-            course_title: 'Title of the course',
+
+            /**
+             * It represents the course title.
```

Comment:
@mattdrayer @asadiqbal08 Is this the text that appears a guidance in the fields?
If yes, I'd like to suggest some edits. We can probably make this more helpful for users.